recipes/scikit-rf/meta.yaml

```diff
@@ -0,0 +1,63 @@
+
+{% set name = "scikit-rf" %}
+{% set version = "0.14.3" %}
+{% set compress_type = "zip" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "3dc7e7aab5e0dd1f5ce692f32b053ec7" %}
+{% set build_num = "0" %}
```

Comment:
I really do not recommend all that `jinja`. Only the version is enough. (Ultimately that is your decision.)